lib/tasks/export.rake

```diff
@@ -0,0 +1,7 @@
+namespace :export do
+  desc 'Generate CSV file with Single Offender IDs paired with NOMIS ids'
+  task offenders: :environment do
+    file = Rails.root.join('lib', 'assets', 'data', 'soi_extract.csv')
```

Comment:
This task has to be run only once, so I would not worry much about these details. In the future the location of the file can be different, but we do not worry about it now and it is nice to see it where it is in the rake task